test/e2e/checknodes.go

```diff
@@ -29,8 +29,7 @@ var _ = Describe("Check nodes", func() {
 
 		nodes, err := Clients.Kubernetes.CoreV1().Nodes().List(metav1.ListOptions{})
 		Expect(err).NotTo(HaveOccurred())
-
-		var nodeCount int64
+		var nodeCount int32
```

Comment:
fyi you can also say `var nodeCount int` and later `Expect(nodeCount).To(BeEquivalentTo(expectedNodeCount))`

https://onsi.github.io/gomega/#beequivalenttoexpected-interface